internal/server/models/deviceflowauthrequest.go

```diff
@@ -8,13 +8,20 @@ import (
 
 type DeviceFlowAuthRequest struct {
 	Model
-	UserCode       string
-	DeviceCode     string
-	Approved       *bool
-	AccessKeyID    uid.ID
-	AccessKeyToken string // to be filled in once approved
-	ExpiresAt      time.Time
+	UserCode   string
+	DeviceCode string
 
-	// can be preloaded if there's an AccessKeyID
-	AccessKey *AccessKey
+	// TODO: remove Approved field? There's no way to reject a request right now
+	// and AccessKeyID != nil indicates approved.
+	Approved    *bool
```

Comment:
Does this sound right? 

If we decide to move the access key creation to the "get request status" part of the flow then we'll need to keep this field, but we'll be able to remove 2 or 3 of the access key fields below.